api/Window.json

```diff
@@ -515,6 +515,69 @@
           }
         }
       },
+      "appinstalled_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/appinstalled_event",
```

Comment:
Oh, wait, I spoke to soon. This is failing CI due to a lack of description.

Reply:
@ddbeck Arrrg. Fixed and passes `yarn test`. Thanks!